gen_sign_cert.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+	echo "Usage: $0 <common name>"
+	exit
+fi
+
+cn=$1
+echo "Generating cert for common name = " ${cn}
+openssl genrsa -out test.key 2048
+
+openssl req -new -text \
+          -extensions v3_req \
+          -addext "subjectAltName = DNS:${cn}" \
+          -subj "/C=US/O=ovnkubernetes/OU=kind/CN=${cn}" \
+          -key test.key \
+          -out test.csr
+
+csr_64=$(cat test.csr | base64 | tr -d "\n")
```

Comment:
`cat` is not necessary

`csr_64=$(<test.csr base64 | tr -d "\n")`

Reply:
Ok

Reply:
it doesnt work like that?

Reply:
err, `base64` takes a file arg, and `-w 0` disables wrapping, so we don't need the shell.

`csr_64=$(base64 -w 0 test.csr)`

Reply:
of course `-w 0` is only GNU coreutils, and doesn't work on busybox or Mac OS X BSD base64, so whatever. :man_shrugging: